PowerPointLabs/PowerPointLabs/Models/PowerPointCurrentPresentationInfo.cs

```diff
@@ -29,7 +29,13 @@ public static Selection CurrentSelection
         {
             get
             {
-                return Globals.ThisAddIn.Application.ActiveWindow.Selection;
+                Selection select = GetSelectionInActiveWindow();
+                // Could be that there is no active window, we can try to activate one
+                if (select == null && ActivateWindow())
+                {
+                    select = GetSelectionInActiveWindow();

```

Comment:
For the case where Ln 32 returns a null for select, I don't think that running GetSelectionInActiveWindow in Ln 36 would change that situation again.

Reply:
Situation will be changed if Line 74 of activating a window works. However, I think i will change this PR to solve a more general problem too when there are no windows available at all (Window count is 0 and there is no active window) so that we can actually test this.

Reply:
Must we forcibly activate a window? Or can we just return null?

The exception happens because we try to get selection from an active window when there is no active window. So what I am saying is that perhaps if there is no active window, instead of getting its selection and catching possible exceptions, just check for the window itself and return null if there is no active window.